Algorithms/Medium/62_UniquePaths/Solution.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def uniquePaths(self, m: int, n: int) -> int:
+        
+        bottomRow = [1]*n
```

Comment:
use snake case and also add spaces between operator and operand, for example: `bottom_row = [1] * n`